legate/core/store.py

```diff
@@ -908,3 +930,55 @@ def find_or_create_partition(self, functor):
         part = converted.construct(self.storage.region, complete=complete)
         self._partitions[functor] = (part, proj)
         return part, proj
+
+
```

Comment:
This class contains all the metadata required for creating the fused op, and extracting the subops from the fused op within the Legate FusedTask. 
All input/output/reduction stores are organized into an array, and prefix sums/offsets are generated, which can be used to index into a particular sub-ops subset of stores